luxtronik/parameters.py

```diff
@@ -1101,7 +1101,7 @@ def __init__(self, safe=True):
             1057: Unknown("ID_Einst_Vorlauf_ZUP"),
             1058: Unknown("ID_Einst_Abtauen_im_Warmwasser"),
             1059: Energy("ID_Waermemenge_ZWE"),
-            1060: Unknown("ID_Waermemenge_Reset"),
+            1060: Energy("ID_Waermemenge_Reset"),
```

Comment:
This is probably the heat amount since the last reset?

Reply:
Yes, I think a reset can be performed via the menu of the heat pump. (So I do not think that this is a reset of the heat pump itself.)

Reply:
Yes, I meant the reset via the menu.